src/components/board/Dashboard.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import Sidebar from '../frontend/Sidebar';
```

Comment:
Add one line above

Reply:
above ?

Reply:
Import npm packages at top of the file, followed by import of custom codes. Leave one line in between these two imports
```js
import React from 'react';
import GoogleLogin from 'react-google-login';
import { withRouter } from 'react-router-dom';

import Sidebar from '../frontend/Sidebar';
import { fetchProjectServices } from '../../services/project';
```